docs/embedded/getting-started/register-api-documentation.md

```diff
@@ -7,12 +7,14 @@ ms.localizationpriority: high
 
 # Register file storage container type application permissions
 
-In order for a SharePoint Embedded application to interact with containers in a consuming tenant, the container type must first be registered in the consuming tenant. Container type registration happens when the owning application invokes the registration API to specify what permissions can be performed against its container type. The registration API also grants access to other Guest Apps to interact with the owning application's containers. For example, a SharePoint Embedded application can grant permissions to another application--a Guest App so that the Guest App can perform backup operations against its containers.
+In order for a SharePoint Embedded application to interact with containers in a consuming tenant, the container type must first be registered in the consuming tenant. Container type registration happens when the owning application invokes the registration API to specify what permissions can be performed against its container type. The registration API also grants access to other guest apps to interact with the owning application's containers. For example, a SharePoint Embedded application can grant permissions to another application--a guest app so that the guest app can perform backup operations against its containers.
 
 Since the registration API controls the permissions that a SharePoint Embedded application can perform against the container in the consuming tenant, this call should be one of the first APIs invoked. Failure to do so results in access denied errors when invoking other APIs against the container and/or the content in the containers.
 
 There are no restrictions on how many times the registration API can be invoked. How often the registration API is invoked and when it's invoked is dependent on the SharePoint Embedded application. However, the last successful call to the registration API determines the settings used in the consuming tenant.
 
+> [!NOTE] : Container type registration may fail if attempted immediately after tenant creation. To avoid issues, wait at least two hours after tenant creation before registering a container type.
```

Comment:
This is formatted incorrectly, the `< [!NOTE]` should be followed by nothing else. Content should be on the next line:
```markdown
> [!NOTE]
> Container type ....
```

Reply:
Also, this is going into the new Registration API doc (Graph docs):
> The registration of a container type in a newly created tenant can fail if the tenant isn't yet fully ready. You might need to wait at least an hour before you can register a container type in a new tenant.

One says up to one hour, and the other says up to two hours. 

Should get in sync on this @javieralvarezchiang

Reply:
We should document what the customer promise is. If there’s a difference in behaviour between the new API and the current registration API, then we should first document the current behaviour in this PR.

When the new APIs go out, we would need to change this whole article to point to the new APIs anyway and would update the customer promise then. @javieralvarezchiang